modules/process/manager.go

```diff
@@ -6,13 +6,12 @@ package process
 
 import (
 	"bytes"
+	"context"
```

Comment:
Personally I've upgraded already, so don't think about me. Just think about random possible contributor staying safe in her latest debian/ubuntu/fedora stable system. We want to reduce the barriers of entry for any contributor.

We should also have Drone test all Go versions we want to support. For comparison Gogs is Travis-tested against Go versions from 1.5 to master.